src/releaseBusV2/release-bus-v2-advance-staging-ref-workflow.test.ts

```
@@ -83,6 +83,11 @@ describe('Release Bus v2 staging ref advancement workflow', () => {
     );
     expect(workflow).toContain('elif [ "$CHECKOUT_OUTCOME" != success ]; then');
     expect(workflow).toContain('failure_phase=checkout');
+    expect(workflow).toContain(
+      'if (.retryable | type) == "boolean" then'
+    );
```

ESLint

[error] Replace `⏎······'if·(.retryable·|·type)·==·"boolean"·then'⏎····` with `'if·(.retryable·|·type)·==·"boolean"·then'`

(prettier/prettier)